templates/weno/layout.html.twig

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>{% block title %} {{ "Welcome"|xlt }} {% endblock %}</title>
+    {{ setupHeader(['jquery-ui', 'jquery-ui-sunny']) }}
```

Comment:
jquery-ui is now illegal to use (same penalty is hanging out on an iceberg for several months). Remove the jquery-ui and use bootstrap instead for styling.

Reply:
This one I don't understand what you mean use bootstrap. There are no examples in the codebase.

![image](https://user-images.githubusercontent.com/834958/77268272-c0d23400-6c7b-11ea-994e-094a4b134adf.png)

Reply:
Bootstrap is loaded automatically and it should look fine just dropping the jquery-ui. So would change above to:
```
{{ setupHeader() }}
```